### PR TITLE
Package Request: mms-git

### PR DESCRIPTION
**Pull request by KenwoodFox**
From: https://aur.archlinux.org/packages/mms-git

Me and my robotics team are hoping we can get this added as a pebuild package!

### CODE REVIEW

File: any/mms-git/cactus.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+nvchecker:
+  - source: aur
+    aur:
```
Comment (petronny) [inline any/mms-git/cactus.yaml:3]: Please also add a `github` source for a git package.

```yaml
  - source: github
    github: mackorone/mms
```

**Review by petronny — CHANGES_REQUESTED** [review]

**Comment by petronny** [comment]
Hi, this package doesn't seem to be an `any` package. Please confirm and if it isn't an `any` package:

1. Move `any/mms-git/cactus.yaml` to `x86_64/mms-git/cactus.yaml`
2. Please contact the maintainer on AUR to set `arch=('x86_64')` in PKGBUILD.
3. If the maintainer doesn't respond, update the `pre_build` in `cactus.yaml` to set `arch=('x86_64')`:
```yaml
pre_build: |
  aur-pre-build
  echo "arch=('x86_64')" >> PKGBUILD
```

**Comment by KenwoodFox** [comment]
> Hi, this package doesn't seem to be an `any` package. Please confirm and if it isn't an `any` package:
> 
>     1. Move `any/mms-git/cactus.yaml` to `x86_64/mms-git/cactus.yaml`
> 
>     2. Please contact the maintainer on AUR to set `arch=('x86_64')` in PKGBUILD.
> 
>     3. If the maintainer doesn't respond, update the `pre_build` in `cactus.yaml` to set `arch=('x86_64')`:
> 
> 
> ```yaml
> pre_build: |
>   aur-pre-build
>   echo "arch=('x86_64')" >> PKGBUILD
> ```

Hey! Thanks for responding, i checked around and i don't *think* there's any reason this couldn't be compiled for other arches? I checked on the aur and the github and everything *seems* to be architecture agnostic, not sure about upstream libs of course.

Regardless, I dont anticipate anybody will complain if this package is x86_64 only so, I'll implement the suggested patch, and thank you!

**Comment by petronny** [comment]
> Hey! Thanks for responding, i checked around and i don't think there's any reason this couldn't be compiled for other arches? I checked on the aur and the github and everything seems to be architecture agnostic, not sure about upstream libs of course.

Hi, to be an `any` package means that its final compiled binary is cross-platform, like the binary built on an x86_64 machine also runs on an ARM board.

If the code is cross-platform but the binary isn't, we should set `arch` to all supported platform like:
```
arch=('x86_64' 'i686' 'aarch64' 'armv6h' 'armv7h')
```

**Comment by KenwoodFox** [comment]
> > Hey! Thanks for responding, i checked around and i don't think there's any reason this couldn't be compiled for other arches? I checked on the aur and the github and everything seems to be architecture agnostic, not sure about upstream libs of course.
> 
> Hi, to be an `any` package means that its final compiled binary is cross-platform, like the binary built on an x86_64 machine also runs on an ARM board.
> 
> If the code is cross-platform but the binary isn't, we should set `arch` to all supported platforms like:
> 
> ```
> arch=('x86_64' 'i686' 'aarch64' 'armv6h' 'armv7h')
> ```

Ah I understand now, i thought any would essentially do that, just make a bin for each major arch. Since I'm unsure about the availability of the upstream qt libs, lets just see if this compiles for arch4edu now? i can implement that suggested change rn if you would like

File: any/mms-git/cactus.yaml (hunk 1 above)
Comment (petronny) [reply any/mms-git/cactus.yaml]: Hi, please also add the `aur` upstream.

**Comment by KenwoodFox** [comment]
Does this build? I dont see its status in the build pages

File: x86_64/mms-git/cactus.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+nvchecker:
+  - source: github
+    github: mackorone/mms
```
Comment (petronny) [inline x86_64/mms-git/cactus.yaml:3]: Hi, please also add the `aur` upstream.
```yaml
nvchecker:
  - source: aur
    aur:
  - source: github
    github: mackorone/mms
```

**Review by petronny — CHANGES_REQUESTED** [review]

File: x86_64/mms-git/cactus.yaml (hunk 2 above)
Comment (KenwoodFox) [reply x86_64/mms-git/cactus.yaml:5]: Ok! Added in 4624f3e1

**Review by petronny — APPROVED** [review]

**Comment by petronny** [comment]
https://github.com/arch4edu/cactus/actions/runs/6524159790/job/17715533270

**Comment by KenwoodFox** [comment]
Wooo yeah!

**Pull request merged**